### PR TITLE
Show PersistentVolumes in StorageClass Detail.

### PR DESCRIPTION
**Pull request by harisanker1780**
The commit on this PR help to list PersistentVolumes created using given StorageClass.

There is one known issue, Searching and Sorting on the PersistentVolume List not working which I am looking into.

preview of this change
screenshot from 2017-10-18 21-49-19

Any suggestions will be greatly appreciated.

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Hi, @harisanker1780. Can you start with signing the CLA?

**Comment by maciaszczykm** [comment]
Looking good. Let us know if you will need any help :)

File: src/app/frontend/storageclass/detail/detail.html [hunk 1]
```diff
@@ -16,4 +16,11 @@
 
 <div layout="column">
   <kd-storage-class-info storage-class="::$ctrl.storageClass"></kd-storage-class-info>
+  <kd-content-card>
+    <kd-content>
+      <kd-persistent-volume-card-list persistent-volume-list="::$ctrl.storageClass.persistentVolumeList"
+                                      persistent-volume-list-resource="::$ctrl.storageClassPersistentVolumesResource">
+      </kd-persistent-volume-card-list>
```
Comment (maciaszczykm) [inline src/app/frontend/storageclass/detail/detail.html]: The issue with sorting is most probably here. Take a look on replica set's pods or events. It should work.

Comment (harisanker1780) [reply src/app/frontend/storageclass/detail/detail.html]: Fixed searching and sorting issue in persistentvolume list.

**Comment by floreks** [comment]
@harisanker1780 I see it's almost finished. Run `gulp serve:prod` and fix all issues. Also we need some tests here, not to decrease coverage.

**Comment by maciaszczykm** [comment]
@harisanker1780 Do you plan to add tests for the backend part?

**Comment by harisanker1780** [comment]
@maciaszczykm Yes, I am working on tests for backend.

**Pull request merged**